### PR TITLE
fix negative price in bundled products

### PR DESCRIPTION
**Pull request by nalezny**
Fix for product bundles extension. Under special circumstances it generates negative price for budled product.

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -95,7 +95,10 @@ Or
 = 1.2.9 =
 * Fixed incorrect order status-pingback handling. 
 
+= 1.2.10 =
+* Fix for product bundles extension. Under special circumstances it generates negative price for budled product.
```
Comment (kavhad) [inline README.txt]: budled should be bundled

File: templates/checkout-page.php [hunk 2]
```diff
@@ -67,6 +67,10 @@
   $shipping_ex_vat = round($shipping_cost, 2);
   $quantity        = $cart_item['quantity'];
 
+	//product bundles extension can generate negative price for one of the bundle products, here is how to fix it
```
Comment (kavhad) [inline templates/checkout-page.php]: Unfortunately this change is based on a false premise. We cannot and should not manipulate price in cart items.
Instead we need to be able to handle negative pricing in our backend.

We need to revert this change.

**Comment by kavhad** [comment]
Hi @nalezny,
Your latest commits reverts all changes. Including those that are about tested version numbers which we want to keep. My only request was to revert the change related to "fix" negative price of bundles in cart, all other changes should be kept.

Sorry if I was not clear about that. Please fix this

**Comment by nalezny** [comment]
Hello @kavhad, I am working on a different approach. When it is done I will submit it again also with a correctly tested version.

**Comment by kavhad** [comment]
@nalezny I've updated the correctly tested version my self in a new PR (PR #42). 

We have multiple plugins and third party integrations, so we won't handle negative prices in the plugins directly as we would need to make the change everywhere instead of handling the negative price correctly in the backend. So I don't see what other approach you had in mind, but I've already spoken to Petter about this.
